### ISSUE TITLE
Problem installing dashydash

### ISSUE THREAD

**Issue by ChristianPresslmayr** [z1c0/dashydash#25]
Hi and Servus!
I found your project and it looks really useful to me - that's exactly what I need I think - really great!

However, when trying to install it on my Raspberry 4, I ran into a problem related to the Node version, which finally results in an error with node-sass.
So, when first installing NodeJS (11.x, the version you mentioned in your Readme), it tells me, that this version is already deprecated. Using newer versions 12 or 14 would be recommended.

When running npm install for dashydash then, it fails with the following error:

gyp ERR! configure error 
gyp ERR! stack Error: ENOENT: no such file or directory, mkdir '/home/pi/dashydash/node_modules/node-sass/.node-gyp'
gyp ERR! System Linux 5.10.92-v7l+
gyp ERR! command "/usr/local/bin/node" "/home/pi/dashydash/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /home/pi/dashydash/node_modules/node-sass
gyp ERR! node -v v11.15.0
gyp ERR! node-gyp -v v8.4.1
gyp ERR! not ok 
Build failed with error code: 1

[...]

pm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@7.0.1 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@7.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

--> So, it seems that installing node-sass version 7.0.1 doesn't work - likely because version 7.0.1 requires Node 12 (see https://github.com/sass/node-sass/releases/tag/v7.0.1)

Of course I also tried installing Node 12, but finally it didn't work either.
Do you have a clue that to do? What would I need to change?

Thanks upfront!
Greetings from Amstetten/AT,
Chris

**Comment by z1c0** [comment]
Hi Christian,

thanks for the feedback - I have not updated this project for quite a while, I guess now is a good time.
Please give me a few days to sort this out. I'd be happy to see another user of `dashydash` :-)

**Comment by ChristianPresslmayr** [comment]
Hi!
Thx for the quick answer!

Meanwhile I solved the installation issue - I found on the internet, that the error is due to some permission problem.
Therefore, I ran the dashydash installation using "npm install --unsafe-perm", then installation was fine.

However, later on, I had to adapt several files (e.g. the gulpfile.js), because the syntax was not correct with the gulp version installed.

Furthermore, I was not really successful in configuring dashydash. When running the dashboard with gulp, I just got a white empty webpage, but no content on it. As there is not really a documentation what to do, and I didn't find out by checking all the files in the project, I finally gave up.

It would be a cool project, but it doesn't work for me.

I can recommend an alternative, which I found then, though: MagicMirror (https://magicmirror.builders/), which is really powerful and highly configurable!

Thx again and sorry for the inconvenience!
Greetings, Chris
